homeassistant/components/sensor/steam_online.py

```diff
@@ -0,0 +1,80 @@
+"""
+homeassistant.components.sensor.steam_online
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Sensor for Steam account status.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.steam_online/
+"""
+from homeassistant.helpers.entity import Entity
+from homeassistant.const import (ATTR_ENTITY_PICTURE, CONF_API_KEY)
+
+ICON = 'mdi:steam'
+
+REQUIREMENTS = ['steamodd==4.21']
+
+
+# pylint: disable=unused-argument
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """ Sets up the Steam platform. """
+    import steam as steamod
```

Comment:
I named it that because I wasn't sure it would mess anything up. For example it used to be called `steam.py` instead of `steam_online.py` but I had to rename it. :-)